index.html

```diff
@@ -0,0 +1,229 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href='https://fonts.googleapis.com/css?family=Lato:400,700' rel='stylesheet' type='text/css'>
+    <link rel="stylesheet" href="css/main_styles.css">
+    <link rel="stylesheet" href="css/animations_transitions.css">
+    <link rel="stylesheet" href="css/game-img-special_effects.css">
+    <link rel="stylesheet" href="css/background_elements.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <title>Microverse Gaming Fest</title>
+</head>
+<body>
+    <main>
+        <header>
+            <nav class="flex-navbar">
+                <div>
+                    <a href="#">
+                        <img class="primary-logo" src="img/MicroverseGamingFestLogo.png" width="10%" alt="MicroverseGamingFestLogo">
+                    </a>
+                </div>
+                <ul class="flex-nav-list">
+                   <li class="list-item" id="homie"><a href="#home-section" class="nav-link">Home</a></li>
+                   <li class="list-item"><a href="#main_event-section" class="nav-link">Main Event</a></li>
+                   <li class="list-item"><a href="#features_games-section" class="nav-link">Featured Games</a></li>
+                   <li class="list-item"><a href="#about_contact-section" class="nav-link">About</a></li>
+                   <li class="list-item"><a href="#partners-section" class="nav-link">Partners</a></li>
+                </ul>
+                <div class="menu">
+                    <div class="menu-line"></div>
+                    <div class="menu-line"></div>
+                    <div class="menu-line"></div>
+                </div>
+            </nav>
+            </header>
+        <section class="home-section" id="home-section">
+            <div class="home-main">
+                <div class="home-title">
+                    <h1 id="home-title">
+                        MICROVERSE GAMING FEST <br>
+                        VIRTUAL GLOBAL EVENT <br>
+                        2023
+                    </h1>
+                </div>
+                <div class="home-parragraf">
+                    <p id="home-parragraf">
+                       Lorem, ipsum dolor sit amet consectetur adipisicing elit.
+                       Sunt aut laborum aliquid, sit ea, numquam itaque architecto.
+                    </p>
+                </div>
+                <div>
+                    <h2 id="home-title2">
+                        2023.07.07(FRI) ~ 09(SUN)
+                    </h2>
+                    <p id="home-parragraf2">
+                        @ Microverse Gaming Room, Microverse Zoom and more
```

Comment:
- [ ] Kindly maintain close to identical fonts for this section on both indexes and about sections. 
- For colors, use the one's provided by the design guide

## Design color:
![image](https://user-images.githubusercontent.com/50990925/233254809-1061e5e9-1940-4125-9cef-2c8f9fb77ed5.png)